### PR TITLE
Change in hamburger icon and handling of left drawer 

### LINKED ISSUE #1159: Change in hamburger icon and handling of left drawer
**Describe the bug**
Inappropriate placement of the hamburger icon

**To Reproduce**

1. Login to talawa admin
2. click on the hamburger button 
3.  see the error

**Expected behavior**
we want to change the UI and placement of the button to make the user experience more intuitive.

**Screenshots**
Screenshot 2023-12-07 at 8 31 12 PM

**Additional details**
Untitled
Look how Docusaurus handles the left menu drawer in [Talawa Docs](https://docs.talawa.io/docs/)

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Vaishali054**
**What kind of change does this PR introduce?**

Change in hamburger icon and handling of left drawer. Changed the UI and placement of the button to make the user experience more intuitive

**Issue Number:**

Fixes #1159 

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

https://github.com/PalisadoesFoundation/talawa-admin/assets/100770875/a8e72a84-b7bc-4591-9b64-7290974809b3

**If relevant, did you update the documentation?**

NA

**Summary**

Change in hamburger icon and handling of left drawer 

**Does this PR introduce a breaking change?**

NA

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

YES

### CODE REVIEW

**Comment by Vaishali054** [comment]
@aashimawadhwa @rishav-jha-mech can you please take a look if possible? Thank you!

**Review by palisadoes — APPROVED** [review]

**Comment by palisadoes** [comment]
@aashimawadhwa @rishav-jha-mech please review and merge if satisfactory

File: src/components/LeftDrawer/LeftDrawer.module.css [hunk 1]
```diff
@@ -104,6 +104,10 @@
   text-transform: capitalize;
 }
 
+.logout {
+  margin-bottom: 50px;
+}
+
```
Comment (noman2002) [inline src/components/LeftDrawer/LeftDrawer.module.css:110]: Why this is defined twice.

File: src/components/LeftDrawer/LeftDrawer.module.css [hunk 2]
```diff
@@ -145,6 +149,10 @@
     z-index: 100;
     animation: openDrawer 0.6s ease-in-out;
   }
+
+  .logout {
+    margin-bottom: 1.5rem !important;
+  }
```
Comment (noman2002) [inline src/components/LeftDrawer/LeftDrawer.module.css:155]: here.

File: src/components/LeftDrawerOrg/LeftDrawerOrg.module.css [hunk 3]
```diff
@@ -148,6 +148,10 @@
   text-transform: capitalize;
 }
 
+.logout {
+  margin-bottom: 50px;
+}
+
```
Comment (noman2002) [inline src/components/LeftDrawerOrg/LeftDrawerOrg.module.css:154]: Here also.

File: src/components/LeftDrawerOrg/LeftDrawerOrg.module.css [hunk 4]
```diff
@@ -189,6 +193,10 @@
     z-index: 100;
     animation: openDrawer 0.6s ease-in-out;
   }
+
+  .logout {
+    margin-bottom: 1.5rem;
+  }
```
Comment (noman2002) [inline src/components/LeftDrawerOrg/LeftDrawerOrg.module.css:199]: here.

**Comment by Vaishali054** [comment]
@noman2002 there are in total 4 logout definitions. We are using two different leftDrawers named LeftDrawer and LeftDrawerOrg. I gave margin-bottom of 50px in dekstop screens so that it doesnt overlap with the close drawer button. The second is for the tablet view as it has a close button at the top instead of at the button therefore the margin-bottom needed to be reduced there. 
The same changes are applied on both LeftDrawers making 4 logout definitions (2 each).

**Comment by noman2002** [comment]
> @noman2002 there are in total 4 logout definitions. We are using two different leftDrawers named LeftDrawer and LeftDrawerOrg. I gave margin-bottom of 50px in dekstop screens so that it doesnt overlap with the close drawer button. The second is for the tablet view as it has a close button at the top instead of at the button therefore the margin-bottom needed to be reduced there. The same changes are applied on both LeftDrawers making 4 logout definitions (2 each).

Thank you for you explanation.

**Review by noman2002 — APPROVED** [review]

**Pull request merged**